drivers/display/display_ili9xxx.c

```diff
@@ -16,18 +16,50 @@ LOG_MODULE_REGISTER(display_ili9xxx, CONFIG_DISPLAY_LOG_LEVEL);
 #include <drivers/spi.h>
 #include <string.h>
 
-#define RES_X_MAX 320U
-#define RES_Y_MAX 240U
+#define ILI9XXX_X_MAX 320U
+#define ILI9XXX_Y_MAX 240U
+#define ILI9XXX_PIX_FMT_MASK \
+	(PIXEL_FORMAT_RGB_888 | PIXEL_FORMAT_RGB_565 | PIXEL_FORMAT_BGR_565)
 
+#define DT_CS_PORT        DT_INST_0_ILITEK_ILI9XXX_CS_GPIOS_CONTROLLER
```

Comment:
It does not belong in this commit, please try to move all of the changes regarding DT and style to one commit or wait for #23245

Reply:
Could you be so kind to explain why? This is done inside of "drivers: display_ili9xxx: obtain display settings from DT"